### PR TITLE
Support multiple VIPs, HAproxy and routing policy

### PR DESCRIPTION
**Pull request by EmilienM**

### CODE REVIEW

File: tearup.yaml [hunk 1]
```diff
@@ -795,7 +795,24 @@
         enabled: yes
         state: restarted
 
-    - name: create the VIP
+    - name: create the VIPs
       shell: |
         #!/bin/bash
-        ip address add {{ vip}} dev lo
+        ip address add {{ item }} dev lo
+        ip rule add from {{ item }} to {{ ansible_facts.default_ipv4.network }}/{{ (ansible_facts.default_ipv4.address + '/' + ansible_facts.default_ipv4.netmask) | ipaddr('prefix') }} lookup 999
```
Comment (mdbooth) [inline tearup.yaml:802]: Do we want to make this idempotent?

File: tearup.yaml [hunk 2]
```diff
@@ -795,7 +795,24 @@
...
+
+    - name: Install HAproxy
+      ansible.builtin.package:
+        name: haproxy
+        state: present
+
+    - name: Put SELinux in permissive mode, logging actions that would be blocked.
+      ansible.posix.selinux:
+        policy: targeted
+        state: permissive
```
Comment (mdbooth) [inline tearup.yaml:818]: What do we need this for? It doesn't look like we're doing anything SELinux is likely to complain about.

If we really do need it, please could you add a comment explaining why.

File: tearup.yaml (hunk 1 above)
Comment (EmilienM) [reply tearup.yaml:802]: yeah I pushed this only in case my laptop goes on fire. I'll re-work it.

**Comment by EmilienM** [comment]
This is horrible. I ended up creating an Ansible role for it. Stay tuned.

**Pull request closed without merging**